src/server/routes/backend-wallet/send-transactions-atomic.ts

```diff
@@ -0,0 +1,151 @@
+import { Type, type Static } from "@sinclair/typebox";
+import type { FastifyInstance } from "fastify";
+import { StatusCodes } from "http-status-codes";
+import type { Address, Hex } from "thirdweb";
+import { insertTransaction } from "../../../shared/utils/transaction/insert-transaction";
+import { AddressSchema } from "../../schemas/address";
+import {
+  requestQuerystringSchema,
+  standardResponseSchema,
+  transactionWritesResponseSchema,
+} from "../../schemas/shared-api-schemas";
+import {
+  maybeAddress,
+  walletChainParamSchema,
+  walletWithAAHeaderSchema,
+} from "../../schemas/wallet";
+import { getChainIdFromChain } from "../../utils/chain";
+import {
+  getWalletDetails,
+  isSmartBackendWallet,
+  type ParsedWalletDetails,
+  WalletDetailsError,
+} from "../../../shared/db/wallets/get-wallet-details";
+import { createCustomError } from "../../middleware/error";
+
+const requestBodySchema = Type.Object({
+  transactions: Type.Array(
+    Type.Object({
+      toAddress: Type.Optional(AddressSchema),
+      data: Type.String({
+        examples: ["0x..."],
+      }),
+      value: Type.String({
+        examples: ["10000000"],
+      }),
+    }),
```

Comment:
Non-blocking: These are the minimal args of a "raw transaction" and may be useful to DRY since it's used in a few places. Then we can add descriptions too.